### DISCUSSION TITLE
USD environment files

### DISCUSSION THREAD

**Discussion by ChristophKin** [abizovnuralem/go2_omniverse#31] [Q&A]
Hi! The USD files of both of your custom environments are several hundred MB. When I try to create my own environment and save it as a USD file, the file is only a few MB. Then when I try to load it into the simulation, I get errors saying that Isaac Sim cannot find any assets and then it loads an empty environment. How did you create your environment USD files? Thanks in advance.

**Comment by abizovnuralem** [comment]
Hello, you need to keep the same path structure as in custom envs, in other cases it will fail to load an environment. For the help, you can watch some videos from nvidia, that uses OpenUSD for it.

**Reply by ChristophKin** [reply]
I will try that, thanks!

**Reply by ChristophKin** [reply]
I have looked at this. But how did you save your environment into one large USD file without dependencies?
